### DISCUSSION TITLE
(Request) Requesting for a Floorp Beta version for desktop users (with different color icon as compared to purple).

### DISCUSSION THREAD

**Discussion by ghost** [Floorp-Projects/Floorp#1436] [Feature Request]
Earlier, I was using Floorp beta alongside Floorp final but somehow I am unable to find the former again. (Request) Requesting for a Floorp Beta version for desktop users (with different color icon as compared to purple). Reason is that I have uninstalled all Firefox browsers (yes, I was using final, dev, nightly and ESR) and now I have Floorp for work and want another Floorp for entertainment. Hope dev allows us to install beta alongside as standalone not as a sub-part like chrome, firefox, edge and opera does with their different versions. Thanks.

**Comment by CutterKnife** [comment]
The official website is still under development and lacks some information, including a link to the beta version.
https://github.com/Floorp-Projects/Floorp/releases/beta

**Reply by ghost** [reply]
Thanks for responding back.

I was referring to this page of Floorp download where beta version used to exist previously but not anymore. I do use beta alongside final. 

![image](https://github.com/user-attachments/assets/72de6f72-c148-4a63-8a36-63c702bba36b)


Can I have it installed as standalone and alongside the Floorp final? And does this beta has different icon so as to recognize visually?

**Reply by CutterKnife** [reply]
Yes, it will be installed as a separate browser called “Floorp Daylight”.

**Reply by ghost** [reply]
Thanks. Do look into my other points and posts for improvement.
